community/caddy/caddy.initd

```diff
@@ -3,11 +3,46 @@
 name="Caddy webserver"
 command="/usr/sbin/caddy"
 command_args="$CADDY_OPTS"
-pidfile="/var/run/caddy.pid"
 command_background=yes
-start_stop_daemon_args="--user caddy --group caddy"
+command_user=caddy:caddy
+extra_commands="checkconfig"
+extra_started_commands="reload"
+
+# Pidfile only required for start-stop-daemon.
```

Comment:
Please keep the singular statement - I will remove it once the move to s-d happens.